src/mdns/mdns.cpp

```diff
@@ -619,5 +633,19 @@ void Publisher::UpdateHostResolutionEmaLatency(const std::string &aHostName, otb
     }
 }
 
+void Publisher::AddAddress(AddressList &aAddressList, const Ip6Address &aAddress)
+{
+    aAddressList.push_back(aAddress);
+}
+
+void Publisher::RemoveAddress(AddressList &aAddressList, const Ip6Address &aAddress)
+{
+    auto it = std::find(aAddressList.begin(), aAddressList.end(), aAddress);
+    if (it != aAddressList.end())
```

Comment:
nit: empty line after declaration
```suggestion
    auto it = std::find(aAddressList.begin(), aAddressList.end(), aAddress);

    if (it != aAddressList.end())
```